.eslintrc.json

```diff
@@ -102,5 +102,7 @@
     "spriteFileLarge": true,
     "icons": true,
     "mapType": true
```

Comment:
There is a `,` missing at the end of line 104

Reply:
Correct in the process of merging develop back into my branch and fixing the merge conflicts.